reports/distributions/liatrio.yaml

```diff
@@ -0,0 +1,6 @@
+name: liatrio
+url: https://github.com/liatrio/liatrio-otel-collector
+maintainers: []
+components:
+    receiver:
+        - gitprovider
```

Comment:
It looks odd to me that a distribution has a single component

Reply:
heh well it might be so much value that's all they need.

Reply:
@adrielp Does liatrio's distro only have one component?

Reply:
Nope @mx-psi, you're spot on. I need to come back and updated this. This is what we include [taken from this manifest.yaml](https://github.com/liatrio/liatrio-otel-collector/blob/main/config/manifest.yaml) (i've updated the distro faster than I could open pr's for sure) 

Extensions:
- zpages
- ballast
- basicauth
- healthcheck
- pprof
- bearertokenauth
- sigv4auth

exporters:
- logging
- otlphttp
- awsxray
- awscloudwatchlogs
- influxdb
- prometheus
- zipkiin
- prometheusremotewrite

processors:
- batchprocessor
- memorylimiter
- resourcedetection
- resource
- k8sattributes
- transform
- filter

receivers:
- otlp
- gitprovider
- prometheus
- kubeletestats
- k8scluster
- jaeger
- zipkin

connectors:
- spanmetrics

Do you want me to make a suggestion to this file to add them?

Reply:
you need to add this to the metadata.yaml of every component you include, and then this tool will pick up the changes and build the report.
This is not blocking this PR.

Reply:
@atoulme  - want me to go ahead and do that now through a separate PR such that it can be picked up through this one?

Reply:
Please open a PR, and we'll see which one lands first.